server/controllers/accountController.js

```diff
@@ -1,15 +1,88 @@
-exports.login = (req, res, next) => {};
+const jwt = require("jsonwebtoken");
+// const { validationResult } = require("express-validator");
 
-exports.signup = (req, res, next) => {};
+const account = require("../models/account");
 
-exports.getAccount = (req, res, next) => {
+exports.login = async (req, res, next) => {
+  const token = req.body.token; // TODO: what token is this? jwt?
+  // User shouldn't have jwt if logging in though. Or is this related to google signin?
+
+  try {
```

Comment:
Let's refactor these blocks to use the asyncHandler middleware

Reply:
Refactored